packages/core/src/components/tabs/tabPanel.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2024 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import classNames from "classnames";
+import * as React from "react";
+
+import { AbstractPureComponent, Classes, Utils } from "../../common";
+
+import { type TabProps } from "./tab";
+import { generateTabIds, type TabTitleProps } from "./tabTitle";
+
+export interface TabPanelProps extends Pick<TabProps, "className" | "id" | "panel">, Pick<TabTitleProps, "parentId"> {
```

Comment:
re `panel` - do we want to inherit the `If omitted, no panel will be rendered for this tab. Can either be an element or a renderer.` behavior?

if so we probably want something similar to https://github.com/palantir/blueprint/pull/6896/files#diff-cf85fdaa862aad83538297ecd6e3a3b6a5a3fee30bd8aaaea4a2294f462c7933R330-R332 where we early return if panel is undefined

Reply:
Yep. I had to remove `TabPanel` from the isomorphic tests then.